test/blackbox-tests/test-cases/melange/merlin.t

```diff
@@ -64,11 +64,12 @@ Dump-dot-merlin includes the melange flags
   B $TESTCASE_ROOT/_build/default/.output.mobjs/melange
   S $TESTCASE_ROOT
   # FLG -ppx '/MELC_COMPILER -as-ppx'
-  # FLG -w @1..3@5..28@30..39@43@46..47@49..57@61..62@67@69-40 -strict-sequence -strict-formats -short-paths -keep-locs
+  # FLG -w @1..3@5..28@30..39@43@46..47@49..57@61..62@67@69-40 -strict-sequence -strict-formats -short-paths -keep-locs -g
```

Comment:
Interesting that `-g` is added by default by dune. Do you think it has any perf implications in Melange, or in other words, is it handled any differently than in OCaml?

Reply:
Melange sets `-g` by default even if not passed, so this is fine.